uint16[] id_list
float64[] current_ma